sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ set(
     inc/azure/storage/common/internal/file_io.hpp
     inc/azure/storage/common/internal/reliable_stream.hpp
     inc/azure/storage/common/internal/shared_key_policy.hpp
+    inc/azure/storage/common/internal/shared_key_policy_lite.hpp
```

Comment:
If this is only used by table, we can move this to table package?

Reply:
done